### PR TITLE
Add docs for features

### PR DESCRIPTION
**Pull request by Ladsgroup**
Part I

### CODE REVIEW

File: revscoring/features/diff.py [hunk 1]
```diff
@@ -31,17 +31,73 @@ def process_chars_added(diff_added_segments):
...
+
+:Returns:
+    float
+
+:Example:
+    ..code_block :: python
+
+        >>> from revscoring.features import diff
+        >>> list(
+            extractor.extract(609079959, [diff.proportion_of_chars_added]))
```
Comment (halfak) [inline revscoring/features/diff.py]: Wow.  This looks wrong.  It should not return a value larger than 1.

Comment (Ladsgroup) [inline revscoring/features/diff.py]: The feature is obviously wrong [1] but I can't understand intention of that feature so I can't fix it.

[1] proportion_of_chars_*added* = chars_*removed* / \

Comment (halfak) [inline revscoring/features/diff.py]: Yikes!  Looks like that should say "chars_added / modifiers.max(revision.chars, 1)"

**Comment by Ladsgroup** [comment]
Something is wrong about my repo, I fixed it and I create a new pull request

**Pull request closed without merging**